pkg/controller/storagecluster/reconcile.go

```diff
@@ -75,6 +80,14 @@ var validTopologyLabelKeys = []string{
 
 var throttleDiskTypes = []string{"gp2", "io1"}
 
+func ensureCreated(obj establisher, r *ReconcileStorageCluster, storagecluster *ocsv1.StorageCluster, reqLogger logr.Logger) error {
```

Comment:
I don't like the use of these functions to call the method of an arbitrary object. Since you loop through all establishers generically anyway, we should just be calling those methods directly.

Reply:
Done